settings.js

```diff
@@ -43,7 +43,13 @@ const defaultSettings = {
     // HTTPS enforcement - whether or not to secure anything and everything sent
     // between the server and client (including web sockets). This is not actually
     // implemented yet.
-    https: {value: 'off', possibleValues: ['on', 'off']}
+    https: {value: 'off', possibleValues: ['on', 'off']},
+
+    // Authorization required - whether or not users will need to be authorized
+    // before they can interact with the server (or view its messages). Anyone
+    // can still register, but an admin must mark the user as authorized before
+    // they will be able to send or receive any information to/from the server.
+    requireAuthorization: {value: 'off', possibleValues: ['on', 'off']}
```

Comment:
Would be nice to have an `authorizationMessage` that appears if you're logged in but not authorized. e.g. 'Contact hello@example.com for an account.'

Reply:
Yeah, good idea!

Reply:
![johndoe](https://user-images.githubusercontent.com/9948030/34332555-cd340962-e929-11e7-8e80-c1b07e53f492.png)